packages/expo-dev-launcher/ios/ReactDelegateHandler/ExpoDevLauncherReactDelegateHandler.swift

```diff
@@ -5,6 +5,7 @@ import EXUpdatesInterface
 
 @objc
 public class ExpoDevLauncherReactDelegateHandler: ExpoReactDelegateHandler, EXDevLauncherControllerDelegate {
+  public weak var rctAppDelegate: RCTAppDelegate?
```

Comment:
nit: maybe add some helper functions than accessing the member directly. e.g. `isReactInstanceValid` or `destroyReactInstance` or even `getRootViewFactory`

Reply:
makes sense, I added `isReactInstanceValid ` and `destroyReactInstance `